gensim/models/word2vec.py

```diff
@@ -185,23 +184,21 @@ def train_sg_pair(model, word, word2, alpha, labels, train_w1=True, train_w2=Tru
         fb = 1. / (1. + exp(-dot(l1, l2b.T)))  # propagate hidden -> output
         gb = (labels - fb) * alpha  # vector of error gradients multiplied by the learning rate
         if train_w1:
-            model.syn1neg[word_indices] += outer(gb, l1)  # learn hidden -> output
```

Comment:
It is a bug, and I just fixed it. 
Just FYI, I have made changes only to the skip gram model with hierarchical sampling